package.json

```diff
@@ -86,6 +86,9 @@
     "@babel/standalone": "7.14.8",
     "@ctrl/tinycolor": "3.3.4",
     "@fluentui/react-icons": "^2.0.154-beta.5",
+    "@griffel/babel-preset": "1.0.0",
+    "@griffel/jest-serializer": "1.0.0",
+    "@linaria/babel-preset": "3.0.0-beta.14",
```

Comment:
This is really silly solution, but I don't see other way to hoist this package up.

### What happens?

Multiple instances of `@linaria/babel-preset` causing build failures as we are overriding static method directly:

https://github.com/microsoft/fluentui/blob/master/packages/make-styles-webpack-loader/src/webpackLoader.ts#L88-L98

(_Linaria loader does the same, but it's not a clear solution_)

![image](https://user-images.githubusercontent.com/14183168/150769944-b7e93597-510a-4b1b-9e03-ee64cfa63cae.png)

`yarn why` reports that the module should be hoisted to root `node_modules`, but this does not happen:

![image](https://user-images.githubusercontent.com/14183168/150769988-37adcc24-a7fb-49b3-b1e4-14d8b2bd49fb.png)

(👆 root `node_modules`)

![image](https://user-images.githubusercontent.com/14183168/150770099-cadfcfc6-4c2d-41f7-96bc-428c82bcef36.png)

(👆 `node_modules` for `@fluentui/webpack-loader-make-styles`)

Reply:
Issue created #21386.